Xamarin.Forms.Platform.iOS/Renderers/BoxRenderer.cs

```diff
@@ -60,5 +65,16 @@ protected override void SetBackgroundColor(Color color)
 
 			SetNeedsDisplay();
 		}
+
+		private void SetCornerRadius()
+		{
+			if (Element == null)
+				return;
+
+			var elementCornerRadius = Element.CornerRadius;
+
+			Layer.MasksToBounds = true;
+			Layer.CornerRadius = (float)elementCornerRadius.TopLeft;
```

Comment:
If I'm understanding the code, this means that on iOS the BoxView will ignore all of the CornerRadius values except for the TopLeft one. Am I missing something?